tests/compatibility-tests/src/main/resources/serial/jbmserial.groovy

```diff
@@ -49,6 +50,11 @@ if (method.equals("write")) {
     topic = new ActiveMQTopic("topic")
     temporary = ActiveMQDestination.createTemporaryQueue("whatever")
     temporaryTopic = ActiveMQDestination.createTemporaryTopic("whatever")
+    if (version.equals("ARTEMIS-SNAPSHOT")) {
+        destination = new ActiveMQDestination("address", "name", ActiveMQDestination.TYPE.DESTINATION, null)
```

Comment:
Same as other comment, for 2.x this really should be using the non "name" constructor this constructor should be deprecated.

Reply:
This is for backwards compatibility. We need to use the field to validate the older format. it's good to be here.

Reply:
sold.